tests/test_mirror_port_span.py

```diff
@@ -0,0 +1,762 @@
+# This test suite covers the functionality of mirror feature in SwSS
```

Comment:
See comment for `test_mirror_port_erspan`, take a look at `dvslib` and let me know if you have any questions!

Reply:
Done. taken care.